### DISCUSSION TITLE
istio 1.22.5 not published to gcr.io

### DISCUSSION THREAD

**Discussion by jwhitcraft** [istio/istio#53222] [General]
Hello,

We ran into a problem on our clusters today where it couldn't pull istio/proxyv2:1.22.5 from gcr.io. It was able to pull from dockerhub just fine.

Are there any plans to get 1.22.5 published to gcr.io?

Thanks

**Comment by howardjohn** [comment]
There was an issue with the release which has been resolved in the meantime. Thanks for the report and your patience. Let us know if you have any other issues
